Makefile

```diff
@@ -2904,6 +2931,43 @@ distclean: clean toolsclean
 dist:
 	git archive --format zip --output $(CLIENTBIN)-$(VERSION).zip HEAD
 
+#############################################################################
+# INSTALL (only for Linux platforms)
+#############################################################################
+
+# Shell scripts for running binaries
+
+$(B)/$(CLIENTBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(CLIENTBIN) "$$@"' >> $@
+
+$(B)/$(SERVERBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(SERVERBIN) "$$@"' >> $@
+
+# Install the .desktop, icon files, license, etc.
+install: release
+ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu" "gnu"))
+	$(echo_cmd) "Installing for Linux platform in $(COPYBINDIR) and $(PREFIX)"
+	@$(INSTALL) -d $(PREFIX)/bin "$(PREFIX)/share/metainfo/" \
```

Comment:
Just a very small nitpick here. I noticed that some paths have "/" at their end and some don't. It does not matter, but it would still be nice to make it consistent.